plugins/outputs/azure_data_explorer/azure_data_explorer_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/influxdata/telegraf"
-	telegrafJson "github.com/influxdata/telegraf/plugins/serializers/json"
+	telegraf_json "github.com/influxdata/telegraf/plugins/serializers/json"
```

Comment:
```suggestion
	json_serializer "github.com/influxdata/telegraf/plugins/serializers/json"
```

Reply:
Changed according to @srebhan suggestion: https://github.com/influxdata/telegraf/pull/15836#discussion_r1743247633